MapboxDirections/MBRoute.swift

```diff
@@ -52,170 +47,22 @@ open class Route: NSObject, NSSecureCoding {
-    @objc open func getCoordinates(_ coordinates: UnsafeMutablePointer<CLLocationCoordinate2D>) {
-        for i in 0..<(self.coordinates?.count ?? 0) {
-            coordinates.advanced(by: i).pointee = self.coordinates![i]
-        }
+    public var routeOptions: RouteOptions {
+        return super.directionOptions as! RouteOptions
     }
     
-    /**
-     An array of `RouteLeg` objects representing the legs of the route.
-     
-     The number of legs in this array depends on the number of waypoints. A route with two waypoints (the source and destination) has one leg, a route with three waypoints (the source, an intermediate waypoint, and the destination) has two legs, and so on.
-     
-     To determine the name of the route, concatenate the names of the route’s legs.
-     */
-    @objc open let legs: [RouteLeg]
-    
-    @objc open override var description: String {
-        return legs.map { $0.name }.joined(separator: " – ")
-    }
-    
-    // MARK: Getting Additional Route Details
-    
-    /**
-     The route’s distance, measured in meters.
-     
-     The value of this property accounts for the distance that the user must travel to traverse the path of the route. It is the sum of the `distance` properties of the route’s legs, not the sum of the direct distances between the route’s waypoints. You should not assume that the user would travel along this distance at a fixed speed.
-     */
-    @objc open let distance: CLLocationDistance
-    
-    /**
-     The route’s expected travel time, measured in seconds.
-     
-     The value of this property reflects the time it takes to traverse the entire route. It is the sum of the `expectedTravelTime` properties of the route’s legs. If the route was calculated using the `MBDirectionsProfileIdentifierAutomobileAvoidingTraffic` profile, this property reflects current traffic conditions at the time of the request, not necessarily the traffic conditions at the time the user would begin the route. For other profiles, this property reflects travel time under ideal conditions and does not account for traffic congestion. If the route makes use of a ferry or train, the actual travel time may additionally be subject to the schedules of those services.
-     
-     Do not assume that the user would travel along the route at a fixed speed. For more granular travel times, use the `RouteLeg.expectedTravelTime` or `RouteStep.expectedTravelTime`. For even more granularity, specify the `AttributeOptions.expectedTravelTime` option and use the `RouteLeg.expectedSegmentTravelTimes` property.
-     */
-    @objc open let expectedTravelTime: TimeInterval
-    
-    /**
-     `RouteOptions` used to create the directions request.
-     
-     The route options object’s profileIdentifier property reflects the primary mode of transportation used for the route. Individual steps along the route might use different modes of transportation as necessary.
-     */
-    @objc open let routeOptions: RouteOptions
```

Comment:
Restore this property as a computed property:

```swift
@objc open let routeOptions: RouteOptions {
    return directionsOptions as! RouteOptions
}
```

and add a similar `matchingOptions` to Match.

Reply:
It's still there, just moved: https://github.com/mapbox/MapboxDirections.swift/pull/236/files#diff-6212178132f11eb7e002f4e0973e9d66R53